hw/top_earlgrey/dv/verilator_sim_cfg.hjson

```diff
@@ -201,6 +201,10 @@
       name: sw_silicon_creator_lib_driver_uart_functest
       sw_images: ["sw/device/silicon_creator/testing/sw_silicon_creator_lib_driver_uart_functest:1"]
     }
+    {
```

Comment:
missing comma before the new entry

Reply:
These are optional in Hjson, but certainly don't hurt.